### ISSUE TITLE
Warning-Spam due to ill-constructed comparison

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [pik-piam/remind#32]
This 
https://github.com/pik-piam/remind/blob/3c91fb8f98ae7e1a29dd5d7d94a94061a6cd5ac1/R/reportTechnology.R#L50-L57
causes a whole lot (5280, to be exact) of 
```
Warning messages:
In if (v_investcost[, t, te] == 0) { ... :
  the condition has length > 1 and only the first element will be used
```

I suggest either explicitly testing only the first region, if that is intended
```R
if (v_investcost[1, t, te] == 0) {
```

or if it is not, testing if all regions have 0 data
```R
if (all(v_investcost[1, t, te] == 0)) {
```

or at least one region has 0 data
```R
if (any(v_investcost[1, t, te] == 0)) {
```

depending on what makes sense for this comparison.

**Comment by Loisel** [comment]
I suggest to be careful here and assing only if `v_investcost == 0`, i.e.,
```
v_investcost[v_investcost == 0] <- pm_inco0_t[v_investcost == 0]
```
There seem to be regional differentiation and zero costs are not meaningful anyways. Opinions @LaviniaBaumstark ?

**Comment by Loisel** [comment]
Addressed in #34
